src/worker/install/install.sh

```diff
@@ -1,7 +1,16 @@
 #!/bin/bash
-
 arch="nvidia"
 
+PUBLISHER_INSTALL=$1
+
+if [ -n "$PUBLISHER_INSTALL" ];
+then
+    if [ $PUBLISHER_INSTALL = true ];
+    then
+        $(dirname "${BASH_SOURCE[0]}")/geneva.sh  $(dirname "${BASH_SOURCE[0]}")/config/geneva_config.json
```

Comment:
do we need to add `sudo` here as we will rm and create soem dir/file under /etc ?